<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FOPositionGui</class>
 <widget class="QWidget" name="FOPositionGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Position</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Orient Camera to:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="leftButton">
         <property name="text">
          <string>Sagittal </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="frontButton">
         <property name="text">
          <string>Front</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="topButton">
         <property name="text">
          <string>Top</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="rotateButton">
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="translateButton">
     <property name="text">
      <string>Translate</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
